addons/my-ocular/status-dot.css

```diff
@@ -7,7 +7,7 @@
   display: inline-block;
 }
 
-#my-ocular-span {
+.sa-ocular-status {
   border-left: 1px solid rgb(204, 204, 204);
```

Comment:
Looks like 2+ line statuses need `display: inline-block;` added here to behave the same way as before

There are more weird things going on though (this change won't be enough), go to CST's profile (https://scratch.mit.edu/users/CST1229/) with "profile page banner" enabled and see the difference between master branch and this branch.
The "profile page banner" (better-featured-project) also has some styles with !important on them which make debugging this more annoying